odbc/src/api/data.rs

```diff
@@ -97,11 +99,13 @@ impl IntoCData for Bson {
     }
 
     fn to_f32(&self) -> Result<f32> {
-        // this is mildly inefficient, unlike converting to i64 and then i32 which is essentially
-        // free (on x86_64 you can literally just change the register address from e.g., RAX to EAX,
-        // so it's literally a no-op), but since mongodb does not actually support 32 bit floats, we would need to do this
-        // conversion somewhere, anyway.
-        Ok(self.to_f64()? as f32)
+        let out = self.to_f64()?;
+        if out > f32::MAX as f64 || out < f32::MIN as f64 {
+            let _ = i64::MAX;
```

Comment:
What does this line do?

Reply:
Nothing. Not sure why I left that in there. Thanks for catching!